common/build.gradle

```diff
@@ -60,6 +60,11 @@ android {
             consumerProguardFiles 'consumer-rules.pro'
         }
     }
+    //Upgrading to Java 8 in order to use YubiKit SDK
+    compileOptions {
```

Comment:
Let's build Authenticator app early with these changes to make sure it is working and so that we don't catch these things during release this time.

Reply:
Or the manifest changes that is...whichever was causing problems with Authenticator

Reply:
We'll be testing with new builds of both Broker and MSAL, which should hopefully fix the manifest mismatch errors.

Reply:
Yes. I'm suggesting that we do that ahead of time rather than waiting until the next month's release time to test this.

Reply:
@shahzaibj  this wont be released next month. we'll cut a private preview build based on next month's release, cherry-pick CBA change, and hand the apk directly to certain customers.

The actual release would be in broker August build.

Reply:
Hmmm...so how do we plan to do the Broker July release? I believe we would release off of `dev` which has the manifest changes and is going to break Authenticator if consumed without consuming a corresponding newer version of MSAL that also consumes the same common. Sounds like we're going to revert again directly on the release branch just like we did for release 5.0.0?

Reply:
@shahzaibj I'll probably just revert my changes on the release branch for the upcoming release. This would probably be easier than taking all the YubiKey changes out of the dev branch in each repo, right?

Reply:
@rpdome Yes, understood. May be I shouldn't have left the feedback here on this PR. The feedback I was providing wasn't about this PR but more generally about all the code changes that have been made to support the YubyKit CBA work. Some of those changes are still in `dev` and @melissaahn reverted them on the `release/5.0.0` branch during the release last time because those changes were breaking Authenticator. @melissaahn is now re-adding those changes to `merge to release` branch on the PR @gitaumoses4 has to merge release release back to dev because otherwise merging that branch back to dev will remove the YubyKit changes from dev as well. From what you're saying it seems like those changes shouldn't have been `dev` to begin with?

Anyway, all I want is to not be in the situation in which we were in during the last release and had to revert changes on the release branch to unblock Broker release. As long as we can solve that problem somehow then we should be fine.

Reply:
In that case... let's revert the change in dev and cherry-pick them into this branch.

Reply:
(We should have around 2 weeks from code complete date to resolve all the possible build issues while we're building the private preview build on the side)